packages/pieces/community/legis/src/lib/actions/create-ticket.ts

```diff
@@ -320,40 +322,47 @@ export const createTicket = createAction({
       {
         params: {
           where: {
-            ticketTypeFormResponseId: numberAdditionalProcess.data[0].id
+            ticketTypeFormResponseId: numberAdditionalProcess.data[0].id,
+            ticketTypeId: 100047
           }
         },
         headers: {
           BearerStatic: context.auth.BearerStatic,
         }
       }
     );
-    const {categoryId, solicitationType} = JSON.parse(`${context.propsValue['catalogItem']}`)
-    const bodyProcess = {
-      "description": 'Prazo compromisso',
-      "body": "",
-      "recipient":  Number(body['Responsavel']) || null,
-      "ticketTypeId": Number(solicitationType),
-      "categoryId": categoryId,
-      "status": "pendente",
-      "assignedTo": 100440,
-      "assignedToUser": Number(body['Responsavel']) || null,
-      "parentTicketId": numberProcess.data.results[0].id,
-      "ticketTypeAdditionalInfo": {
-        ...body,
-        "forma_de_abertura": "Automática"
+    try {
+      const {categoryId, solicitationType} = JSON.parse(`${context.propsValue['catalogItem']}`)
+      const bodyProcess = {
+        "description": 'Prazo compromisso',
+        "body": "",
+        "recipient":  Number(body['Responsavel']) || null,
+        "ticketTypeId": Number(solicitationType),
+        "categoryId": categoryId,
+        "status": "pendente",
+        "assignedTo": 100440,
```

Comment:
Mesma coisa do que comentei acima.

const ASSIGNED_TO_-nome_definido- = 100440

```suggestion
        "assignedTo": ASSIGNED_TO_-nome_definido-,
```